### ISSUE TITLE
Test failure on Windows

### ISSUE THREAD

**Issue by seandavi** [seandavi/GEOquery#17]
Unfortunately I don't have a build report to point you to because our windows devel build didn't happen today (for an unrelated reason) but I noticed that on windows GEOquery fails a unit test:

1 Test Suite : 
GEOquery RUnit Tests - 14 test functions, 0 errors, 1 failure
FAILURE in testSuppFileSupport: Error in checkEquals(10, ncol(fres)) : Mean relative difference: 0.3

Test files with failing tests

   test_SuppFileSupport.R 
     testSuppFileSupport 

Basically it's because file.info() returns a different number of columns on Windows (7) than on unix (10).

So the test should be fixed.

Thanks,
Dan
